docs/environment.md

```diff
@@ -86,6 +86,49 @@ It sets the `PATH` and some more environment variables. But more importantly it
 An example of this would be the [`libglib_activate.sh`](https://github.com/conda-forge/glib-feedstock/blob/52ba1944dffdb2d882d824d6548325155b58819b/recipe/scripts/activate.sh) script.
 Thus, just adding the `bin` directory to the `PATH` is not enough.
 
+## Traditional `conda activate`-like activation
+
+If you prefer to use the traditional `conda activate`-like activation, you could use the `pixi shell-hook` command.
+
+```shell
+$ which python
+python not found
+$ eval "$(pixi shell-hook)"
+$ (default) which python
+/path/to/project/.pixi/envs/default/bin/python
+```
+
+!!! warning
+    It is not encouraged to use the traditional `conda activate`-like activation, as deactivating the environment is not really possible. Use `pixi shell` instead.
+
+### Using `pixi` with `direnv`
+
+This allows you to use `pixi` in combination with `direnv`.
+Enter the following into your `.envrc` file:
+
+```shell title=".envrc"
+pixi install # (1)!
+eval "$(pixi shell-hook)" # (2)!
+```
+
+1. This ensures that your pixi environment is always up to date when you enter the directory.
+2. This activates the environment. `direnv` ensures that the environment is deactivated when you leave the directory.
+
+```shell
+$ cd my-project
+direnv: error /my-project/.envrc is blocked. Run `direnv allow` to approve its content
+$ direnv allow
+direnv: loading /my-project/.envrc
+✔ Project in /my-project is ready to use!
```

Comment:
It is a bit annoying that this is always displayed when entering the directory. You can mitigate this by using `pixi install 2>/dev/null` but this leads to the installation loading bar to be hidden. Ideally there would be a cli flag `pixi install -q` that still displays a loading bar during installation but doesn't output the `Project in ... is ready to use!`

Wdyt?

Reply:
Wouldnt you expect a -q to also hide the progressbars?

Reply:
Yes, that's the problem :/

Reply:
Alternatively we could print the text to stdout instead of stderr..?

Reply:
When we skip the `pixi install` part, the message is only displayed when something actually changed 🤩